File: 프로그래머스/Level2/피보나치 수.py
# https://programmers.co.kr/learn/courses/30/lessons/12945

def solution(n):
    answer = []
    for i in range(n+1):
        if i==0 or i==1:
            answer.append(i)
        else:
            temp = answer[i-1] + answer[i-2]
            answer.append(temp%1234567)
    return answer[-1]

File: 프로그래머스/Level2/피보나치 수.js
// https://programmers.co.kr/learn/courses/30/lessons/12945

const solution = (n) => {
	const fibo = [0, 1];
	for (let i = 2; i <= n; i++) {
		fibo.push((fibo[i - 2] + fibo[i - 1]) % 1234567);
	}
	return fibo[fibo.length - 1];
};
